types/defines/workflows.d.ts

```diff
@@ -3,7 +3,7 @@
  * that makes a Workflow instance fail immediately without triggering a retry
  */
 declare module "cloudflare:workflows" {
-  export abstract class NonRetryableError extends Error {
+  export class NonRetryableError extends Error {
     /**
      * `__brand` is used to differentiate between `NonRetryableError` and `Error`
```

Comment:
While this is getting updated, why is this a jsdoc comment? Users don't need to see it

Reply:
good point, i'll change it into a normal one